docs/gatsby-config.js

```diff
@@ -5,6 +5,12 @@ const path = require('path')
 const remarkSlug = require('remark-slug')
 
 module.exports = {
+  flags: {
+    DEV_SSR: true,
+    FAST_DEV: true,
```

Comment:
this is experimental, right?

Reply:
yes. gatsby gave me a message that i should try these out...so i'm trying them out :)